tasks/dist/css

```diff
@@ -0,0 +1 @@
+node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
```

Comment:
This is true for the case where you call the bin via package.json. Not sure
if it will work here since you call it in a js file. I'll try it when im
home and let you know.

On Jul 20, 2016 23:19, "Mark Otto" <notifications@github.com> wrote:

> In tasks/dist/css
> <https://github.com/twbs/bootstrap/pull/20332#discussion_r71598457>:
>
> > @@ -0,0 +1 @@
> > +node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
>
> In https://docs.npmjs.com/cli/run-script, it sounded like if the
> devDependency is there, npm run will do that for us.
>
> In addition to the shell's pre-existing PATH, npm run adds
> node_modules/.bin to the PATH provided to scripts. Any binaries provided by
> locally-installed dependencies can be used without the node_modules/.bin
> prefix. For example, if there is a devDependency on tap in your package,
> you should write:
>
> Reading that, maybe I just need to add node-sass to devDependency? Again,
> I've no idea so I'm likely wrong 😁.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/twbs/bootstrap/pull/20332/files/795f6d924b1c1eb9b637917a41f2c83655408c97#r71598457>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtdykpzNATYk8LkXMda5OCPTJJQIlks5qXoLrgaJpZM4JRHjn>
> .
>